src/rustdoc-json-types/lib.rs

```diff
@@ -45,12 +45,19 @@ pub struct ExternalCrate {
 /// question, or can be used by a tool that takes the json output of multiple crates to find
 /// the actual item definition with all the relevant info.
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
-pub struct ItemSummary {
+pub struct ExternalItem {
     /// Can be used to look up the name and html_root_url of the crate this item came from in the
     /// `external_crates` map.
     pub crate_id: u32,
-    /// The list of path components for the fully qualified path of this item (e.g.
-    /// `["std", "io", "lazy", "Lazy"]` for `std::io::lazy::Lazy`).
+    /// The rustdoc list of path components for the fully qualified path of this item
+    /// (e.g. `["std", "io", "lazy", "Lazy"]` for `std::io::lazy::Lazy`).
+    ///
+    /// Note that there are multiple different possible and valid path for each item, the one here
+    /// is currently implementation defined.
```

Comment:
This doc improvement is great. Until this can be resolved fully, could you submit just this as a stand alone PR that doesn't change the format?

Reply:
Okay, but may I ask why ? And which part exactly you want me to move to another PR (the whole new doc, only the note or something else) ?

Reply:
Just the note I think. I'd rather have a new PR because the discussion here is about a format change, and it would be confusing looking back on the PR if this ended up being just a doc improvement.

Reply:
Done. I ended up just leaving the disclaimer.